package.json

```diff
@@ -18,7 +18,7 @@
     "lint:markdown": "markdownlint *.md content/**/*.md",
     "lint:css": "stylelint \"themes/opentermsarchive/assets/css/*.css\"",
     "lint:js": "eslint themes/opentermsarchive/assets/js/",
-    "start:dev": "hugo serve --watch --verbose --disableFastRender",
+    "start:dev": "hugo serve --watch --logLevel debug --disableFastRender",
```

Comment:
When I try locally I have this: `Error: unknown flag: --logLevel`

Reply:
It's probably linked to your hugo version because `--logLevel` was added to [v0.114.0](https://github.com/gohugoio/hugo/releases/tag/v0.114.0). Could you please check it?

Reply:
You're right. Could you specify in the README that hugo >= v0.114.0 is required please?